profiles/base.yaml

```diff
@@ -115,6 +115,7 @@ dump_junk: false
 dump_item_count: 9
 use_barb_combos: false
 whirlwind_trainables:
+aiming_trainables:
```

Comment:
Do you have a YAML with an example of this setting?

Reply:
ah yes!